crates/swc/tests/tsc-references/parser.numericSeparators.decimal_es2015.1.normal.js

```diff
@@ -1,14 +1,14 @@
-1_000_000_000;
-1.1_00_01;
-1e1_0;
-1e+1_0;
-1e-1_0;
-1.1e10_0;
-1.1e+10_0;
-1.1e-10_0;
-12_34_56;
-1_22_333;
-1_2.3_4;
-1_2.3_4e5_6;
-1_2.3_4e+5_6;
-1_2.3_4e-5_6;
+1000000000;
+1.10001;
+10000000000;
+10000000000;
+0.0000000001;
+11000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
```

Comment:
I prefer to keep changes to a minimum.
Maybe it's better to just remove the numeric separator `_`.

```diff
- 11000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+ 1.1e100;
```